frontend/components/App/App.tsx

```diff
@@ -93,6 +95,22 @@ const App = ({ children, location }: IAppProps): JSX.Element => {
   // We will do a series of API calls to get the data that we need to display
   // warnings to the user about various token expirations.
 
+  useQuery(["android_enterprise"], () => mdmAndroidAPI.getAndroidEnterprise(), {
+    ...DEFAULT_USE_QUERY_OPTIONS,
+    retry: false,
+    enabled:
+      false && // TODO: reenable when the BE is completed
+      !!isGlobalAdmin &&
+      !!config?.mdm.android_enabled_and_configured &&
+      config?.android_enabled, // TODO: remove android feature flag
+    onSuccess: () => {
+      setAndroidEnterpriseDeleted(false);
+    },
+    onError: () => {
+      setAndroidEnterpriseDeleted(true);
+    },
```

Comment:
Should this depend on a particular status code (e.g., 404) rather than any error?